tools/ci.sh

```diff
@@ -259,26 +261,8 @@ function ci_qemu_arm_build {
     make ${MAKEOPTS} -C ports/qemu-arm CFLAGS_EXTRA=-DMP_ENDIANNESS_BIG=1
     make ${MAKEOPTS} -C ports/qemu-arm clean
     make ${MAKEOPTS} -C ports/qemu-arm test
-    make ${MAKEOPTS} -C ports/qemu-arm BOARD=sabrelite test
-}
-
-########################################################################################
-# ports/qemu-riscv
-
-function ci_qemu_riscv_setup {
-    ci_gcc_riscv_setup
-    sudo apt-get update
-    sudo apt-get install qemu-system
-    qemu-system-riscv32 --version
-}
-
-function ci_qemu_riscv_build {
-    make ${MAKEOPTS} -C mpy-cross
-    make ${MAKEOPTS} -C ports/qemu-riscv submodules
-    make ${MAKEOPTS} -C ports/qemu-riscv
-    make ${MAKEOPTS} -C ports/qemu-riscv clean
-    make ${MAKEOPTS} -C ports/qemu-riscv -f Makefile.test submodules
-    make ${MAKEOPTS} -C ports/qemu-riscv -f Makefile.test test
+    make ${MAKEOPTS} -C ports/qemu-arm BOARD=SABRELITE test
+    make ${MAKEOPTS} -C ports/qemu-arm BOARD=VIRT_RV32 test
```

Comment:
Is it worth keeping these split into two jobs? The advantages I can see are runtime (can be concurrent) and extra context when debugging (can see immediately if something fails on ARM and/or RISC-V, independently of each other).

Reply:
Yeah, that would be a good idea, to keep them separate.

The main motivation to combine them is to only need to run one setup step.  At the moment that takes about 45 seconds, but I guess if we split it then the ARM one only needs to install the ARM toolchain, and RV32 one only needs to install the RISC-V toolchain.  Then the only thing in common is installation of qemu, which is probably very fast.

Reply:
Now done, two separate CI jobs.